packages/issue-utils/src/issue-utils.ts

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// eslint-disable-next-line node/no-extraneous-import
+import {Octokit} from '@octokit/rest';
+import {GCFLogger, logger as defaultLogger} from 'gcf-utils';
+
+export interface Issue {
```

Comment:
Not a big deal, but is there a reason to have our own interface instead of the type defined in one of the Octokit suites?
Maybe add a comment about it?

Reply:
I'll add a comment in a follow up. The reason is because relying on the third-party times can be prone to fragility. Additionally their types can be weird (e.g. `labels: (string | {name?: string})[];`) which forces you to write more defensive code. We coerce the types to make it easier to work with.